modules/aleph/src/Aleph/AlephRequest.php

```diff
@@ -0,0 +1,380 @@
+<?php
+
+namespace Drupal\aleph\Aleph;
+
+/**
+ * Class AlephRequest.
+ *
+ * Entity with setters/getters for Z37 (REQUESTS) in Aleph.
+ *
+ * The Z37 contains requests information.
```

Comment:
You gotta loose all these references to Z37. It's the job of your API to *hide* all the complexity of how things is working behind the scenes, so don't throw it all in the face of the API user.